tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "allowJs": true,
+        "checkJs": true,
+        "strict": true,
+        "module": "esnext",
+        "moduleResolution": "node"
+    }
+}
```

Comment:
Missing line break

Reply:
Oops. I normally have lint fix this type of thing automatically on save.